test/test_transforms.py

```diff
@@ -1834,6 +1764,83 @@ def test_pad_with_mode_F_images(self):
         assert_equal(padded_img.size, [edge_size + 2 * pad for edge_size in img.size], check_stride=False)
 
 
+def _test_randomness(fn, trans, configs):
```

Comment:
This is fine as-is for me. If we want, we can go one step further about parametrization here. I'm thinking of somthing like this:


```py
@pytest.mark.skipif(stats is None, reason="scipy.stats not available")
@pytest.mark.parametrize('fn, trans, conf', [
    (F.invert, transforms.RandomInvert, {}),
    (F.posterize, transforms.RandomPosterize, {"bits": 4}),
   ...

])
@pytest.mark.parametrize('p', (.5, .7))
def test_randomness(fn, trans, conf, p):
```

and we could get rid of all the other functions and the 2 for loops inside. WDYT?

Again this is optional: feel free to leave as-is if you prefer ;)

Reply:
Yeah, I think, we can parametrize it. I have made the changes.